### PR TITLE
feat(account-lib): transaction and transaction builder tests

### PR DESCRIPTION
**Pull request by gibrangul95**
cc @fdmirza @nimraarif21

### CODE REVIEW

File: modules/account-lib/package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "prepublishOnly": "npm run compile",
     "prepare": "npm run gen-protobuf && npm run gen-protobufts &&  npm run hedera-gen-protobuf && npm run hedera-gen-protobufts && tsc && cp -r ./resources ./dist",
     "unit-test": "nyc -- mocha --opts test/mocha.opts \"test/unit/**/*.ts\"",
+    "test-xrp": "nyc -- mocha --opts test/mocha.opts \"test/unit/coin/xrp/**/*.ts\"",
```
Comment (fdmirza) [inline modules/account-lib/package.json:26]: remove this

File: modules/account-lib/src/coin/xrp/transaction.ts [hunk 2]
```diff
@@ -83,12 +83,14 @@ export class Transaction extends BaseTransaction {
...
-    const txJSON = JSON.stringify(this._xrpTransaction);
-    this._signedTransaction = initApi().sign(txJSON, keys);
+    if (keyPair.length > 0) {
+      const keys = {
+        privateKey: keyPair[0].getKeys().prv,
+        publicKey: keyPair[0].getKeys().pub,
+      };
+      const txJSON = JSON.stringify(this._xrpTransaction);
+      this._signedTransaction = initApi().sign(txJSON, keys);
+    }
```
Comment (fdmirza) [inline modules/account-lib/src/coin/xrp/transaction.ts:93]: Do we want to throw error if no keypairs provided?

**Review by fdmirza — COMMENTED** [review]
looks good.

File: modules/account-lib/package.json (hunk 1 above)
Comment (gibrangul95) [reply modules/account-lib/package.json:26]: Planned on removing it when we squash commits before making the Bitgo PR

File: modules/account-lib/src/coin/xrp/transaction.ts (hunk 2 above)
Comment (gibrangul95) [reply modules/account-lib/src/coin/xrp/transaction.ts:93]: No, otherwise the buildImplementation would fail in TransactionBuilder

**Pull request merged**